### ISSUE TITLE
Warnings in CPython master

### ISSUE THREAD

**Issue by tirkarthi** [PavanTatikonda/itypes#11]
I tried testing Django Rest Framework on CPython master and the test suite passes but while running the tests I came across the below warning. Since CPython master branch is the one that will be used for Python 3.8 running tests might cause `ImportError` in future. Possible solution to ensure compatibility : https://github.com/pyparsing/pyparsing/pull/16/files

```
/home/karthi/drf-venv/lib/python3.8/site-packages/itypes.py:2: 
DeprecationWarning: Using or importing the ABCs from 'collections' instead of 
from 'collections.abc' is deprecated, and in 3.8 it will stop working
```

Thanks much for DRF :)

**Comment by tomchristie** [comment]
Okay, thanks! Will aim to have dropped this as an optional dependency by then.

**Comment by PavanTatikonda** [comment]
@tirkarthi  Took the transfer of this repo from Tom and released 1.2.0 to PyPI which has the fix for collections deprecation warning
